docs/sources/whatsnew/whats-new-in-v9-3.md

```diff
@@ -0,0 +1,244 @@
+---
+_build:
+  list: false
+aliases:
+  - /docs/grafana/latest/guides/whats-new-in-v9-3/
+description: Feature and improvement highlights for Grafana v9.3
+keywords:
+  - grafana
+  - new
+  - documentation
+  - '9.3'
+  - release notes
+title: What's new in Grafana v9.3
+weight: -33
+---
+
+# What’s new in Grafana v9.3 (Beta)
+
+Welcome to Grafana 9.3. Read What's New to learn all about the latest release. For more detail, refer to the [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
+
+## New navigation
+
+Available in **beta** in all editions of Grafana
+
+Use Grafana’s redesigned navigation to get full visibility into the health of your systems, by quickly jumping between features as part of your incident response workflow.
+
+As Grafana has grown from a visualization tool to an observability solution, we’ve added many tools along the way. This has resulted in pages that are visually inconsistent or hard to find. This nav update gives Grafana a new look and feel, and makes page layouts and navigation patterns more consistent.
+
+We’ve revamped the navigation menu and grouped related tools together, making it easier to find what you need. Pages in Grafana now leverage new layouts that include breadcrumbs and a sidebar, allowing you to quickly jump between pages. We’ve also introduced a header that appears on all pages in Grafana, making dashboard search accessible from any page.
+
+To try out Grafana’s new navigation, enable the `topnav` feature toggle. If you are a Cloud Advanced customer, open a ticket with our support team and we will enable it for you.
+
+**Note:** The Grafana documentation has not yet been updated to reflect changes to the navigation.
+
+## View dashboards in Spanish, French, German, and Simplified Chinese
+
+Generally available in all editions of Grafana
+
+We have added 4 new languages to Grafana: Spanish, French, German and Simplified Chinese.
+
+With millions of users across the globe, Grafana has a global footprint. In order to make it accessible to a wider audience, we have taken the first steps in localizing key workflows. You can now set Grafana’s language for the navigation, viewing dashboards, and a handful of settings. This will cover the main activities a Viewer would perform within Grafana.
```

Comment:
```suggestion
With millions of users across the globe, Grafana has a global footprint. In order to make it accessible to a wider audience, we have taken the first steps in localizing key workflows. You can now set Grafana’s language for the navigation, viewing dashboards, and some settings. This will cover the main activities a Viewer performs within Grafana.
```